server/gc/src/main/java/org/apache/accumulo/gc/GarbageCollectionAlgorithm.java

```diff
@@ -312,25 +327,28 @@ public void collect(GarbageCollectionEnvironment gce) throws TableNotFoundExcept
       } finally {
         candidatesSpan.end();
       }
-      deleteBatch(gce, batchOfCandidates);
+      totalBlips += deleteBatch(gce, batchOfCandidates);
```

Comment:
This may count the same blip multiple times.  This loop can take a subset of the candidates and then loop over all of the blips.  So if there are 5 subsets of candidates and 100 total blips in the metadata table then it may return 500 instead of 100.  Could just take the last count.

```suggestion
      totalBlips = deleteBatch(gce, batchOfCandidates);
```

Reply:
Ah Ok. I will do a follow up fix.